### PR TITLE
Make README less booty

### PR DESCRIPTION
**Pull request by spencerhyphengold**
Readme is now as riveting as The Da Vinci Code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
...
+npm run dev
+```
+
+## Contributing
+All issues are kept under the `Projects` tab. If you find anything you'd like to
+add or see fixed, add it there. 
+
+If you'd like to contribute, create a new branch and make changes as you'd like. 
+Pull requests should to be reviewed by at least one active contributor. 
+Please include screenshots with any frontend changes.
```
Comment (SujeethJinesh) [inline README.md]: Add in:

To make a pull request:
1) Make a branch off of `master` by doing `git checkout -b <branch name>`
2) Make your changes and then do `git add .`, `git commit -m "I made xyz changes"`, `git push --set-upstream origin <branch name>`
3) Go back to github and you'll see an option to open up a PR.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+
+Don't break prod pls.
```
Comment (SujeethJinesh) [inline README.md]: Add in:

If you want to test changes that could break prod, we will use a "staging" site that mimics production but won't affect our live site. It is ok if this site is down or breaks as it is used for testing.

1) Log in to Heroku: https://dashboard.heroku.com/
2) Go to bright-futures-academy-staging
3) Go to `deploy`
4) Scroll to the bottom, you should see "Deploy a GitHub branch"
5) Select the branch you'd like to deploy
6) If you want to see logs you can run `heroku login`  and put in the credentials and then `heroku logs --tail --app bright-futures-academy-staging` or see the logs in the console
7) If everything goes well you can see your app on https://bright-futures-academy-staging.herokuapp.com/

File: README.md [hunk 3]
```diff
@@ -1,2 +1,38 @@
...
+## Running the app
+You can start a development server like this:
+```
+npm run server
+```
+Or you can run the full app like this:
+```
+npm run dev
+```
+
```
Comment (SujeethJinesh) [inline README.md]: If you want to just run the front end then `cd client` and `npm start`.

Comment (SujeethJinesh) [reply README.md]: Oh also don't forget to mention running `npm install`

Comment (spencerhyphengold) [reply README.md]: `npm run postinstall` covers this

**Pull request merged**